### PR TITLE
MGMT-3370: Enable KubeAPI deployment from test-infra

### PR DESCRIPTION
**Pull request by RazRegev**
Added the required dependencies to Skipper build container.

### CODE REVIEW

File: Dockerfile.test-infra [hunk 1]
```diff
@@ -2,6 +2,16 @@ FROM quay.io/ocpmetal/assisted-service:latest AS service
 
 FROM quay.io/centos/centos:8.3.2011
 
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.15
```
Comment (tsorya) [inline Dockerfile.test-infra:5]: not svc please, why we need it?

Comment (filanov) [reply Dockerfile.test-infra:5]: Do you prefer to that another image?

**Comment by filanov** [comment]
@RazRegev please add readme.

File: Dockerfile.test-infra [hunk 2]
```diff
@@ -22,5 +22,16 @@ COPY --from=service /clients/assisted-service-client-*.tar.gz /build/pip/
 RUN pip3 install --no-cache-dir -I -r /tmp/requirements.txt && \
   pip3 install --upgrade /build/pip/*
 
+FROM golang:1.15
```
Comment (tsorya) [inline Dockerfile.test-infra:25]: docker.io is a bad option too. let push it to quay.io or atleast regular ci

**Comment by RazRegev** [comment]
closed, decided to with https://github.com/openshift/assisted-test-infra/pull/528

**Pull request closed without merging**